Configuration/TCA/Module/Options.php

```diff
@@ -85,6 +85,12 @@
                 'type' => 'check',
             ],
         ],
+        'auto_orient_custom' => [
```

Comment:
I get the problem you face, but I tested locally and I don't face it (I'm using GraphicsMagick, not ImageMagick).

Are you using ImageMagick? If so, what about something more "targeted" at ImageMagick, in the naming for instance, with something like `fix_im_auto_orient` for instance?

Reply:
Yes, can be done, i leave that up to you :-)